sm6125.c

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright (c) 2022, Linaro Limited */
+
+#include <sys/mman.h>
+#include <err.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "debugcc.h"
+
+static struct debug_mux cpu_cc = {
+	.phys = 0xf111000,
+	.size = 0x1000,
+	.block_name = "cpu",
+
+	.enable_reg = 0x1c,
+	.enable_mask = BIT(0),
+
+	.mux_reg = 0x1c,
+	.mux_mask = 0x3ff << 8,
+	.mux_shift = 8,
+
+	.div_reg = 0x1c,
+	.div_mask = 0xf << 28,
+	.div_shift = 28,
+	.div_val = 8,
```

Comment:
Hmm it seems sdm845 doesn't set this here, but uses `measure_clk::fixed_div` (typically omitted parameter) in the array below... haven't checked the code but might make a difference?

Reply:
once again, could you please check?

Reply:
```
sony-pdx201 ~ $ sudo ./debugcc -p sm6125 -a -b cpu
                                        perfcl_clk: 184.388192MHz (184388192Hz)
                                         pwrcl_clk: 119.631488MHz (119631488Hz)
```

And a bit later:

```
sony-pdx201 somainline # ./debugcc -p sm6125 -a -b cpu
                                        perfcl_clk: 37.496800MHz (37496800Hz)
                                         pwrcl_clk: 225.600832MHz (225600832Hz)
```

Just like APCS on SM8250 these reads seem random and fluctuate often: https://github.com/andersson/debugcc/pull/16#discussion_r1080662248